frontend/dockerui/config.go

```diff
@@ -493,6 +458,21 @@ func (bc *Client) IsNoCache(name string) bool {
 	return false
 }
 
+func (bc *Client) DockerIgnorePatterns(ctx context.Context) ([]string, error) {
+	if bc == nil {
+		return nil, nil
+	}
+	bctx, err := bc.buildContext(ctx)
+	if err != nil {
+		return nil, err
+	}
+	if !bctx.localBuild {
```

Comment:
just `bctx.context == nil` iiuc

Reply:
It didn't seem like taking this approach worked -- `context` seemed to be set to `nil` even in the case of a local build.

Reply:
`bctx.context == nil` means no remote context.